.msggen.json

```diff
@@ -330,7 +330,6 @@
         "NewaddrAddresstype": {
             "all": 2,
             "bech32": 0,
-            "p2sh-segwit": 1,
```

Comment:
Technically we could leave these in, as a trace for future reference, so we don't end up reusing the same number for a new enum variant. But I'll merge this as is, since that's out of scope for this PR.